### PR TITLE
$ARTIFACTS_DIR  is set by platform

### PR DESCRIPTION
**Pull request by anandrkskd**
Signed-off-by: anandrkskd <anandrkskd@gmail.com>

**What type of PR is this?**
/kind bug

**What does does this PR do / why we need it**: This will fix the report generation on periodic jobs 

**Which issue(s) this PR fixes**:

Fixes #?

**PR acceptance criteria**:

- [ ] Unit test 

- [ ] Integration test 

- [ ] Documentation 

- [x] I have read the [test guidelines](https://github.com/openshift/odo/blob/master/docs/dev/test-architecture.adoc)

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

**Comment by prietyc123** [comment]
@anandrkskd I am not sure if the Artifact dir is exposed for multi test env because it has not been captured https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_odo/4399/pull-ci-openshift-odo-master-unit/1356478367305568256#1:build-log.txt%3A25

File: scripts/openshiftci-presubmit-unittests.sh [hunk 1]
```diff
@@ -5,8 +5,7 @@ set -e
 # show commands
 set -x
 
-export ARTIFACTS_DIR="/tmp/artifacts"
-export CUSTOM_HOMEDIR=$ARTIFACTS_DIR
+export CUSTOM_HOMEDIR=$ARTIFACT_DIR
```
Comment (mik-dass) [inline scripts/openshiftci-presubmit-unittests.sh:8]: We also need to change the env below at https://github.com/openshift/odo/blob/9d246db05bc5cf434e3cc74e88e10e0e47b66c03/scripts/openshiftci-presubmit-unittests.sh#L21

File: scripts/openshiftci-presubmit-all-tests.sh [hunk 2]
```diff
@@ -18,8 +18,7 @@ make bin
 mkdir -p $GOPATH/bin
 make goget-ginkgo
 export PATH="$PATH:$(pwd):$GOPATH/bin"
-export ARTIFACTS_DIR="/tmp/artifacts"
-export CUSTOM_HOMEDIR=$ARTIFACTS_DIR
+export CUSTOM_HOMEDIR=$ARTIFACT_DIR
```
Comment (mik-dass) [inline scripts/openshiftci-presubmit-all-tests.sh:21]: You need to change the env below too

File: scripts/openshiftci-periodic-tests.sh [hunk 3]
```diff
@@ -11,8 +11,7 @@ make bin
 mkdir -p $GOPATH/bin
 make goget-ginkgo
 export PATH="$PATH:$(pwd):$GOPATH/bin"
-export ARTIFACTS_DIR="/tmp/artifacts"
-export CUSTOM_HOMEDIR=$ARTIFACTS_DIR
+export CUSTOM_HOMEDIR=$ARTIFACT_DIR
```
Comment (mik-dass) [inline scripts/openshiftci-periodic-tests.sh:14]: same as https://github.com/openshift/odo/pull/4399/files#r568413016

File: scripts/openshiftci-presubmit-unittests.sh (hunk 1 above)
Comment (prietyc123) [reply scripts/openshiftci-presubmit-unittests.sh:8]: I think we don't need an extra env var CUSTOM_HOMEDIR for the assignment, instead we can directly use ARTIFACTS_DIR where ever we need.

**Review by prietyc123 — CHANGES_REQUESTED** [review]

**Comment by anandrkskd** [comment]
@prietyc123 We should not remove CUSTOM_HOMEDIR env. I gave a quick look in the code and it is used in code AFAIU.

**Comment by prietyc123** [comment]
> We should not remove CUSTOM_HOMEDIR env. I gave a quick look in the code and it is used in code AFAIU.

Yes, but we can remove in the code as well.

**Comment by anandrkskd** [comment]
I guess then It should be done on another PR right?

**Pull request merged**